apps/api/src/services/larc.ts

```diff
@@ -1,10 +1,10 @@
 import type { database } from "@packages/db";
-import { eq } from "@packages/db/drizzle";
-import type { z } from "zod";
-
+import { type SQL, eq, getTableColumns, gte, ilike, lte, or } from "@packages/db/drizzle";
 import { and } from "@packages/db/drizzle";
 import { larcSection, websocCourse } from "@packages/db/schema";
-import type { larcQuerySchema, larcSectionSchema } from "../schema/larc.ts";
+import { isTrue } from "@packages/db/utils";
+import type { z } from "zod";
+import type { larcQuerySchema, larcResponseSchema, larcSectionSchema } from "../schema/larc.ts";
```

Comment:
Unrelated to this change (actually because I'm bad I didn't catch it in the initial LARC PR) but consider exporting `larc.ts` in `apps/api/src/schema/index.ts` so you can just import from `$schema`.